index.html

```diff
@@ -2236,6 +2236,10 @@ <h2>Service Endpoints</h2>
       <pre class="example nohighlight" title="Various service endpoints">
 {
   "service": [{
+    "id":"did:example:123#linked-domain",
+    "type": "LinkedDomains",
```

Comment:
Is this a real use case? Are we going to define "LinkedDomain"? If so, I'm good... if not, let's use a real example.